### PR TITLE
nvme: sanitize: Allow sanitize action string values

### PR DESCRIPTION
**Pull request by ikegami-t**
Signed-off-by: Tokunori Ikegami <ikegami.t@gmail.com>

### CODE REVIEW

File: nvme.c [hunk 1]
```diff
@@ -4588,6 +4588,28 @@ static int ns_rescan(int argc, char **argv, struct command *cmd, struct plugin *
 	return err;
 }
 
+static int parse_sanact(char *str, __u8 *val)
+{
+	int len = strlen(str);
+
+	if (!strncasecmp(str, "exit-failure", len > 1 ? len : 1))
```
Comment (igaw) [inline nvme.c:4595]: Couldn't also accept the old accepted values here? E.g. Also the documentation should be updated accordingly

**Review by igaw — CHANGES_REQUESTED** [review]
at least the documentation needs to be updated.

File: nvme.c (hunk 1 above)
Comment (ikegami-t) [reply nvme.c:4595]: By the change both the old accepted values and new string values are accepted. Noted I will update the documentation also.

**Comment by ikegami-t** [comment]
Updated the documentation.

**Pull request merged**